tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -107,10 +108,19 @@ public void AcquireTokenTest()
         {
             TestCommon.ResetInternalStaticCaches();
 
-            _mobileTestHelper.AcquireTokenTestHelper(
-                _xamarinController,
-                LabUserHelper.GetDefaultUserAsync().GetAwaiter().GetResult(),
-                CoreUiTestConstants.AcquireTokenInteractive);
+            try
+            {
+                _mobileTestHelper.AcquireTokenTestHelper(
+                    _xamarinController,
+                    LabUserHelper.GetDefaultUserAsync().GetAwaiter().GetResult(),
+                    CoreUiTestConstants.AcquireTokenInteractive);
+            }
+            catch (FileLoadException ex)
+            {
+#pragma warning disable CA2201 // Do not raise reserved exception types
```

Comment:
Please try to avoid these directives that disable best practices. You can read about them on msdn and they usually give you a pretty good idea on how to proceed. In this case, the solution is to throw an `InvalidOperationException` or some other more focused exception.